include/macros.h

```diff
@@ -71,6 +71,8 @@
 
 #define MODE_NTSC 0
 #define MODE_MPAL 1
-#define MODE_PAL 2
+#define MODE_PAL  2
+
+#define FORCE_CRASH { *(vs8*)0 = 0; }
```

Comment:
why'd you move this here? types.h is explicitly included _everywhere_ so it'd be easier to access there no?